AGENTS.md

```
@@ -183,120 +183,117 @@ src/
 
 ### Relay Server
 | Command | Description |
 |---------|-------------|
 | `gssh relay start` | Start relay server |
-| `gssh relay authorize <pubkey>` | Authorize a machine on relay |
-| `gssh relay revoke <id>` | Revoke machine authorization |
-| `gssh relay machines` | List authorized machines |
-| `gssh relay trusted` | List trusted relays |
-| `gssh relay untrust <url>` | Remove relay from trusted list |
+| `gssh relay access add <gssh-user:...>` | Grant relay membership |
+| `gssh relay access remove <user-id|label>` | Revoke relay membership |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)